src/handlers/news.ts

```diff
@@ -0,0 +1,33 @@
+import { Message } from "whatsapp-web.js";
+import * as cli from "../cli/ui";
+import config from "../config";
+import Parser from 'rss-parser';
+
+// Init parser from rss-parser
+let parser = new Parser();
+
+const handleMessageNEWS = async (message: Message, prompt: string) => {
+	try {
+		cli.print(`[RSS-NEWS] Received prompt from ${message.from}: ${prompt}`);
+	
+		let feed = await parser.parseURL(config.newsRSSURL);
+		message.reply(feed.title);
+
+		let counter = 0;
+		feed.items.forEach(item => {
+			if (config.maxRSSItems != counter) {
+				message.reply(item.title + ': ' + item.link);
+				counter++;
+			} else {
+				return;
+			}
+		});
```

Comment:
Something like this will make it simpler and easier to read

```suggestion
		feed.items.slice(0, config.maxRSSItems -1).forEach(item => {
			message.reply(item.title + ': ' + item.link);
		});
```